tutorials/intro-to-geospatial-analysis/Sample.js

```diff
@@ -0,0 +1,132 @@
+//  Geometries
+Map.setCenter(-88, 41.8, 9);
+function multi(feature) {
+ //  Reduce number of vertices in geometry; the number is to specify maximum error in meters
+ var simple = feature.simplify(1000);
+ //  Find centroid of geometry
+ var center = simple.centroid();
+ //  Create buffer around geometry; the number represents the width of buffer in meters
+ var buff = center.buffer(500);
+ //  return variable from function
+ return buff;
+}
+//  Load Feature Collection of the dissolved boundary of Chicago stored in assets
+var Chic = ee.FeatureCollection('users/tirthankar25/chicago_diss');
+//  Load Feature Collection of the neighborhood-scale Chicago data stored in assets
+//  var Chic_coll=ee.FeatureCollection('users/tirthankar25/chicago');
+//  Map function over Feature Collection
+//  var Res=Chic_coll.map(multi);
+//  Add the layer to the map
+// Map.addLayer(Res);
+Map.addLayer(Chic, {}, 'Chicago dissolved');
+//  Find the rectangle that emcompasses the southernmost, westernmost, easternmost, and northernmost
+//  points of the feature
+//  var bound = Chic.geometry().bounds();
+//  Find the polygon covering the extremities of the feature
+//  var convex = Chic.geometry().convexHull();
+//  Map.addLayer(bound, {},'Chicago bounds');
+//  Map.addLayer(convex, {},'Chicago convex hull');
+//  Find the area common to two or more features
+//  var Intersect=bound.intersection(convex,100);
+//  Map.addLayer(Intersect, {},'Bound and convex intersection');
+//  Find the area encompassing two or more features; number is the maximum error in meters
+//  var Union=bound.union(convex,100);
+//  Map.addLayer(Union, {},'Bound and convex union');
+//  Find area of feature
+//  var Ar=Chic.geometry().area();
+//  Find length of line geometry
+//  var length=Chic.geometry().length();
+//  Find permeter of feature
+//  var peri=Chic.geometry().perimeter();
+//  print (peri)
+//  Features
+//  Create Geometry
+var varGeometry = ee.Geometry.Polygon(0, 0, 40, 30, 20, 20, 0, 0);
+//  Create Feature from Geometry
+var varFeature = ee.Feature(varGeometry, {
+ Name: ['Feature name', 'Supreme'],
+ Size: [500, 1000]
+});
+//  Get values of a property
+//  var Arr=varFeature.get('Size');
+//  print (Arr);
+//  Select a subset of properties and rename them
+var varFeaturenew = varFeature.select(['Name'], ['Descriptor']);
+print(varFeaturenew);
+//  Images
+Map.setCenter(-88, 41.8, 9);
+var Raw = ee.ImageCollection('MODIS/006/MYD11A2');
+var ROI = ee.FeatureCollection('users/tirthankar25/chicago');
+print(Raw);
+var bandsel = Raw.select(0);
+//  var bandsel=Raw.select('LST_Day_1km');
+//  var filtered=Raw.filterDate('2002-12-30','2004-4-27');
+//  print(filtered);
+//  var limited=Raw.limit(50);
+//  print(limited);
+//  print(bandsel);
```

Comment:
Either remove commented out code or uncomment.

Reply:
Uncommented and made things run sequentially.